### DISCUSSION TITLE
How can I simulate pressing `M-x`?

### DISCUSSION THREAD

**Discussion by g-gundam** [lem-project/lem#1549] [Q&A] [answered]
## Context

I want `SPC SPC` to be the equivalent of hitting `M-x` in my [Space Menu for Lem](https://github.com/lem-project/lem/discussions/1508).  This is another Spacemacs behavior I'd like to replicate.

## The Emacs Way

In Emacs, I can run:

```lisp
(general-simulate-key "M-x")
```

This creates an interactive function `general-simulate-M-x`, and calling it results in Emacs behaving as if I pushed `M-x` myself.  

## What's the Lem equivalent of this?

**Comment by vindarel** [comment, marked as answer]
(lem:execute-command nil)

not sure about why nil

**Reply by g-gundam** [reply]
That works.  Thank you.
